src/recipe/Baked.Recipe.Service.Application/Domain/DomainServicesModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Reflection;
+
+namespace Baked.Domain;
+
+public class DomainServicesModel
+{
+    public List<Assembly> References { get; init; } = [];
+    public List<string> Usings { get; init; } = [];
+    public List<ServiceModel> Services { get; init; } = [];
+}
```

Comment:
lets make this `DomainServiceCollection : List<ServiceModel>`

and still have `References` and `Usings` props

:thinking: